src/presentation/device.rs

```diff
@@ -26,13 +44,29 @@ use std::collections::BTreeMap;
 use std::num::ParseIntError;
 use uuid::Uuid;
 
+/// Initialisation state.
```

Comment:
We use both the US and UK versions for `initialization` and `initialisation` interchangeably. 

Likely a result of different internationalization settings from computer to computer. For example, the UK spelling shows a spell checker error, whereas the US version does not.

Thoughts on standardizing around US spellings in favor of keeping the same conventions consistent?

Reply:
just need to take into account if the lib is used? as this will be a breaking change so we should need to increase the major version

Reply:
Yeah, that's a good point. There are some downloads on crates.io, so someone may be using it. We could add an alias method and #[deprecated] tag, but my sense is that this work isn't worth adding to this PR.

Reply:
Agree, let's have @cobward opinion on this.

Reply:
Our convention is to use US spelling, however I wrote this code before we decided on that convention. I think we can have a separate task to update the code and comments to use US spelling, and to add [`typos-cli`](https://crates.io/crates/typos-cli) to the CI to enforce that.